app/src/main/java/com/example/zuzanna/budgetplanner/DatabaseHandler.java

```diff
@@ -92,17 +92,17 @@ public float getTotalSum() {
     }
 
     //todo parameterise
-    public Map<String, String> getLastSevenDays() {
+    public Map<String, String> getLastDays(int daysBefore) {
         // Generate dates for last week period Unix format
         Calendar cal = new GregorianCalendar();
         long today = cal.getTimeInMillis();
-        cal.add(Calendar.DAY_OF_MONTH, -7);
-        long sevenDaysAgo = cal.getTimeInMillis();
+        cal.add(Calendar.DATE, daysBefore);
```

Comment:
daysBefore here has to be a negative number to get the past week instead of the next one